src/sentry/static/sentry/app/components/assistant/guideAnchor.tsx

```diff
@@ -214,8 +214,8 @@ type WrapperProps = {disabled?: boolean} & Props;
 export default class GuideAnchorWrapper extends React.Component<WrapperProps> {
   render() {
     const {disabled, children, ...rest} = this.props;
-    if (disabled) {
-      return children;
+    if (disabled || window.localStorage.getItem('hide_anchors') === '1') {
```

Comment:
We have a `utils/localStorage`, but... questionable if it's still needed

Reply:
Where does hide_anchors get set to true?

Reply:
Agree with Mark, `hide_anchors` is not set to `'1'` in this PR, nor is there a reference to `hide_anchors` on `master`.

Reply:
@markstory @dashed It gets set here: https://github.com/getsentry/demo-add-on/blob/main/src/marketo/inject.jsx#L6

Reply:
@scefali IMO, it'll be better to expose an API from this external add-on and attach it to `window`. And then control it from here.

Reply:
@dashed hmmm like a variable on `window?? Why is that better?

Reply:
@scefali something like `window.demoAddOn.hideAnchors()`.

Reply:
@dashed and that would update localstorage underneath?

Reply:
Not sure I agree, I think having a local storage check makes sense. Does it matter where it's set?